conans/client/conf/__init__.py

```diff
@@ -13,12 +13,12 @@
 default_settings_yml = """
 # Only for cross building, 'os_build/arch_build' is the system that runs Conan
 os_build: [Windows, WindowsStore, Linux, Macos, FreeBSD, SunOS]
-arch_build: [x86, x86_64, ppc32, ppc64le, ppc64, armv6, armv7, armv7hf, armv7s, armv7k, armv8, armv8_32, armv8.3, sparc, sparcv9, mips, mips64, avr]
+arch_build: [x86, x86_64, ppc32be, ppc32, ppc64le, ppc64, armv6, armv7, armv7hf, armv7s, armv7k, armv8, armv8_32, armv8.3, sparc, sparcv9, mips, mips64, avr, sh4le]
 
 # Only for building cross compilation tools, 'os_target/arch_target' is the system for
 # which the tools generate code
-os_target: [Windows, Linux, Macos, Android, iOS, watchOS, tvOS, FreeBSD, SunOS, Arduino]
-arch_target: [x86, x86_64, ppc32, ppc64le, ppc64, armv6, armv7, armv7hf, armv7s, armv7k, armv8, armv8_32, armv8.3, sparc, sparcv9, mips, mips64, avr]
+os_target: [Windows, Linux, Macos, Android, iOS, watchOS, tvOS, FreeBSD, SunOS, Arduino, Neutrino]
+arch_target: [x86, x86_64, ppc32be, ppc32, ppc64le, ppc64, armv6, armv7, armv7hf, armv7s, armv7k, armv8, armv8_32, armv8.3, sparc, sparcv9, mips, mips64, avr, sh4le]
```

Comment:
Can you please add Little Endian arch for QNX targets , like : aarch64le , and armv7le?

Reply:
I think currently available ARM targets are already representing little endian ARM

Reply:
In general the ARM architectures can use both BE and LE. And sometimes a mix.
The QCC compiler specifically state the the endianess for the ARM architectures.
e.g.:
QNX 7.0 and 6.5 -> armv7le
QNX 7.0 ->aarch64le

Reply:
@Johnnyxy Thanks for the article,I think it would be nice to secify them in the arch_target at least for readability and complience with the qcc target names

Reply:
No problem. I want QNX support too. :)

*_target settings
-----------------
@zmostafa you did not say for what you use the ```arch_target``` setting but be aware that you should not use the ```*_target``` settings in conan for normal compilation for you destination systems.
As the [documentation](https://docs.conan.io/en/latest/systems_cross_building/cross_building.html?highlight=gnu%20triplet#gnu-triplet-convention) states you should use the ```*_target``` settings only if you compile a software (os, arch) that compiles something for another target (os_target, arch_target) target.

If you compile a compiler on your build machine (Windows; x86_64) which runs on Linux (Linux; x86) that compiles for QNX (Neutrino; aarch64le) then your settings would look like this:
```python
os_build = "Windows"        # os that builds the compiler
arch_build = "x86_64"       # arch that builds the compiler
os = "Linux"                # os that runs the compiler
arch = "x86"                # arch that runs the compiler
os_target = "Neutrino"      # os that the built compiler should compile for
arch_target = "aarch64le"   # arch that the built compiler should compile for
```

If you compile only a piece of software (application, library) that runs on QNX (Neutrino, aarch64le) then you just use ```os``` and ```arch``` (maybe with os_build, arch_build _if nessescary_).
```python
os_build = "Windows"        # os that builds the application/library
arch_build = "x86_64"       # arch that builds the application/library
os = "Neutrino"             # os that runs the application/library
arch = "aarch64le"          # arch that runs the application/library
```

My recipes contains additionally to ```os``` and ```arch``` the ```*_build``` settings ```os_build```, ```arch_build``` because I have to distinguish between the QNX SDP for Windows and Linux (```os_build```).

Reply:
@Johnnyxy Thanks for the clarification :) That means  "arch" line in the settings.yml has to be extended with QNX arch?

I am just building the applications, I have normal make files and Cmake and just want to use conan with those files.

Reply:
Yes, exactly. If you just build libraries and applications then you have to set ```os = "Neutrino"```/```arch = "aarch64le"``` for your target system. I did this wrong in the beginning and had to do some refactoring because of my misunderstanding of the ```*_target``` settings.

> As general rule: ```*_target``` is almost never needed in the daily work of the most developers.

But some need this for special embedded boards or other rare systems or "simply" developing compilers so conan gives us this feature.